docs/advanced/document-store/index.md

```diff
@@ -133,6 +134,203 @@ da = DocumentArray(
 
 Using dataclass gives you better type-checking in IDE but requires an extra import; using dict is more flexible but can be error-prone. You can choose the style that fits best to your context.
 
+
+## Search summary
+
+DocArray supports multiple storage backends with different search features. The following table showcases relevant functionalities that are supported (✅) or not supported (❌) in DocArray depending on the backend:
+
+
+| Name                                                           | Construction                             | fast vector search | vector search + filter | filter|
+|----------------------------------------------------------------|------------------------------------------|---------------|------------------------|---------------|
+| [`None`](./../../../fundamentals/documentarray/#documentarray) | `DocumentArray()`                        | ✅             | ✅                      | ✅             |
```

Comment:
On second thought about the word "fast", does in memory docarray use some ANN technique or is it exhaustive search?